code/modules/mob/living/simple_animal/hostile/bosses/paperwizard.dm

```diff
@@ -0,0 +1,167 @@
+	usage_probability = 40
+	boss_cost = 30
+	boss_type = /mob/living/simple_animal/hostile/boss/paper_wizard
+	needs_target = FALSE
+	say_when_triggered = "Rise, my creations! Jump off your pages into this realm!"
+
+/datum/action/boss/wizard_summon_minions/Trigger()
+	if(..())
+		var/list/minions = list(
+		/mob/living/simple_animal/hostile/stickman,
+		/mob/living/simple_animal/hostile/stickman/ranged,
+		/mob/living/simple_animal/hostile/stickman/dog)
+		var/list/directions = cardinal.Copy()
+		for(var/i in 1 to 3)
+			var/minions_chosen = pick_n_take(minions)
+			new minions_chosen (get_step(boss,pick_n_take(directions)))
+
+
+//Mimic Ability
+//Summons mimics of himself with magical papercraft
+//Hitting a decoy hurts nearby people excluding the wizard himself
+//Hitting the wizard himself destroys all decoys
+/datum/action/boss/wizard_mimic
+	name = "Craft Mimicry"
+	button_icon_state = "mimic_summon"
+	usage_probability = 30
+	boss_cost = 40
+	boss_type = /mob/living/simple_animal/hostile/boss/paper_wizard
+	say_when_triggered = ""
+
+/datum/action/boss/wizard_mimic/Trigger()
+	if(..())
+		var/mob/living/target
+		if(!boss.client) //AI's target
+			target = boss.target
+		else //random mob
+			var/list/threats = boss.PossibleThreats()
+			if(threats.len)
+				target = pick(threats)
+		if(target)
+			var/mob/living/simple_animal/hostile/boss/paper_wizard/wiz = boss
+			var/directions = cardinal.Copy()
+			for(var/i in 1 to 3)
+				var/mob/living/simple_animal/hostile/boss/paper_wizard/copy/C = new (get_step(target,pick_n_take(directions)))
+				wiz.copies += C
+				C.original = wiz
+				C.say("My craft defines me, you could even say it IS me!")
+			wiz.say("My craft defines me, you could even say it IS me!")
+			wiz.forceMove(get_step(target,pick_n_take(directions)))
+			wiz.minimum_distance = 1 //so he doesn't run away and ruin everything
+			wiz.retreat_distance = 0
+		else
+			boss.atb.refund(boss_cost)
+
+/mob/living/simple_animal/hostile/boss/paper_wizard/copy
+	desc = "tis a ruse!"
+	health = 1
+	maxHealth = 1
+	alpha = 200
+	boss_abilities = list()
+	melee_damage_lower = 1
+	melee_damage_upper = 5
+	minimum_distance = 0
+	retreat_distance = 0
+	ranged = 0
+	loot = list()
+	var/mob/living/simple_animal/hostile/boss/paper_wizard/original
+
+//Hit a fake? eat pain!
+/mob/living/simple_animal/hostile/boss/paper_wizard/copy/adjustHealth(amount)
+	if(amount > 0) //damage
+		if(original)
+			original.minimum_distance = 3
+			original.retreat_distance = 3
+			original.copies -= src
+			for(var/c in original.copies)
+				qdel(c)
+		for(var/mob/living/L in range(5,src))
+			if(L == original || istype(L, type))
+				continue
+			L.adjustBruteLoss(50)
+		qdel(src)
+	else
+		..()
+
+//Hit the real guy? copies go bai-bai
+/mob/living/simple_animal/hostile/boss/paper_wizard/adjustHealth(amount)
+	if(amount > 0)//damage
+		minimum_distance = 3
+		retreat_distance = 3
+		for(var/copy in copies)
+			qdel(copy)
+			..()
```

Comment:
this should be . = ..() and also have two less tabs so it doesn't happen once per copy and actually permits admin-forced healing